### ISSUE TITLE
Venue Button - Directness

### ISSUE THREAD

**Issue by GantMan** [infinitered/ChainReactApp2023#85]
Little nit on the UI.  When I click "Venue" I get "Conference and workshops venue" title.

That's A and B.  But the address shown is B then A.  Meaning it shows the workshop address.

ALSO:  Will this show the correct one on the correct dates?

I'd suggest a more direct UI, perhaps it's a bad spot for that cute slider.

**Comment by mazenchami** [comment]
@JenFucci @justinhuskey thoughts on this ticket?

**Comment by justinhuskey** [comment]
I had the same thought as Gant. It seemed weird to see workshops under "Venue." I thought I would see the Armory. It might make sense to forego a slider and stack them for clarity.

**Comment by JenFucci** [comment]
We can stack them vs. putting them in a slider since the sponsors will move to another screen.  

Full-width like the static section from the explore screen, keeping the content for each the same as it is now in the slider: 

<img width="258" alt="example section from explore screen" src="https://user-images.githubusercontent.com/17892823/229878935-a3f4e9fa-44d6-438c-b5a1-8d0647b00083.png">

**Comment by JenFucci** [comment]
I updated this since it was just copying over the styles. Can the screen be called "venues" (plural) so we can list all locations relevant to attendees? This also includes the Expensify office for the afterparty. 

On figma: https://www.figma.com/file/gvx9aFkqmXfXtvdDBRQwne/mobile-app?node-id=1908%3A5047&t=2pPytaoPfZ2Ygi5b-1

**Comment by jamonholmgren** [comment]
"Venues" makes sense to me @JenFucci.

**Comment by mazenchami** [comment]
@GantMan we're going to move the "Sponsors" to the "Info" tab. That tab currently has "Hosted by the IR Team" and then "Code of Conduct". Where should the "Sponsors" list live? first? second? third? i would assume the sponsors would say first?

**Comment by GantMan** [comment]
can we accordion those sections?  no need to care about ordering too much if we can.

**Comment by mazenchami** [comment]
sounds good, i'll look into an accordion experience. @JenFucci let me know if you have any thoughts here

**Comment by JenFucci** [comment]
@mazenchami & @GantMan, what about rows for each section after the IR intro linking to another screen? The sponsors and code of conduct section are a bit lengthy.

Added on Figma here: 
https://www.figma.com/proto/gvx9aFkqmXfXtvdDBRQwne/mobile-app?page-id=51%3A753&node-id=1919-5196&viewport=-618%2C375%2C0.57&scaling=min-zoom

**Comment by GantMan** [comment]
Yup.  That's good too.  I like either: accordion or just subnav like Jenna has in figma.
